### PR TITLE
Remove APY

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Remove APY skills, agent, and services that will belong to a separate repo.
> **:warning: Do not merge if the new repo https://github.com/valory-xyz/apy is not prepared first.**

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [x] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: autonomy/deploy/generators/docker_compose/templates.py [hunk 1]
```diff
@@ -90,7 +90,6 @@
     container_name: abci{node_id}
     image: {runtime_image}
     environment:
-      - PYTHONHASHSEED=0
```
Comment (DavidMinarsch) [inline autonomy/deploy/generators/docker_compose/templates.py:93]: I think we should keep this for all our envs

File: autonomy/deploy/generators/kubernetes/templates.py [hunk 2]
```diff
@@ -251,8 +251,6 @@
             value: "1"
           - name: LOG_FILE
             value: "/logs/aea_{validator_ix}.txt"
-          - name: PYTHONHASHSEED
```
Comment (DavidMinarsch) [inline autonomy/deploy/generators/kubernetes/templates.py:254]: see above

File: tox.ini [hunk 3]
```diff
@@ -74,9 +72,7 @@ deps =
     {[deps-base]deps}
 passenv = *
 extras = all
-setenv =
```
Comment (DavidMinarsch) [inline tox.ini:77]: please leave this, see above

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

**Comment by DavidMinarsch** [comment]
> @dvilelaf `numpy`'s usage is now very minimal and could potentially be removed:
> 
> https://github.com/valory-xyz/open-autonomy/blob/d03ee5d320e3ac8eff13bca85d393289ff4f2454/packages/valory/connections/abci/tests/test_fuzz/base.py#L42-L50
> 
> @angrybayblade same for `pandas`:
> 
> https://github.com/valory-xyz/open-autonomy/blob/d03ee5d320e3ac8eff13bca85d393289ff4f2454/autonomy/analyse/benchmark/aggregate.py#L61-L111

Suggest opening an issue for this and resolving in a future PR.

File: autonomy/deploy/generators/docker_compose/templates.py (hunk 1 above)
Comment (Adamantios) [reply autonomy/deploy/generators/docker_compose/templates.py]: Reverted change in 25abec60251a55b9d5e1a763c5b9be6290e8eef8.

File: autonomy/deploy/generators/kubernetes/templates.py (hunk 2 above)
Comment (Adamantios) [reply autonomy/deploy/generators/kubernetes/templates.py]: Reverted change in 25abec60251a55b9d5e1a763c5b9be6290e8eef8.

File: tox.ini (hunk 3 above)
Comment (Adamantios) [reply tox.ini]: Reverted change in 25abec60251a55b9d5e1a763c5b9be6290e8eef8.

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**